docs/guides/becoming-a-cna-as-an-open-source-org-or-project.md

```diff
@@ -0,0 +1,221 @@
+
+This guide was written for Open Source organizations and projects that are interested in becoming a CNA and managing their own CVEs. This is not a guide for becoming a [Root](https://www.cve.org/ResourcesSupport/AllResources/CNARules#section_3_roots) which has additional requirements.
+
+This document will help cover the following topics:
+
+* Benefits of becoming a CNA as an Open Source project
+* Does your organization or project meet the requirements to become a CNA?
+* What is the process for becoming a CNA?
+
+A future document will cover other topics about CNA operations.
+
+## Why become a CNA?
+
+CVEs are a common way to record and share details about vulnerabilities in software projects. Because of their near-ubiquity, CVEs are often the method that downstream consumers will use to know when software they're using is vulnerable and requires remediation. This is especially evident with their use of security scanners and tools that report findings discovered in software and systems using the CVE identifiers.
+
+CVE Numbering Authorities (CNAs) are the entities that can allocate CVE IDs and create corresponding CVE Records within a particular scope (a product, a project or group of affiliated projects, and the like). Becoming a CNA means that your organization will have a defined scope and will be able to create CVE IDs and Records for that defined scope autonomously.
+
+Below are some of the benefits of becoming a CNA:
+
+* **Provide high-quality authoritative CVE Records for your users.** CVE Records created by third-parties can be incomplete or inaccurate. 
+* **CVEs can't be issued for projects in a CNA's scope without first reporting to the CNA.** This means that reporters _must_ initially engage with your CNA, thus reducing confusion and allowing subject-matter experts on the project and security policy to weigh in on whether to create a CVE for a given disclosure.
+* **Assign CVE IDs without needing to share embargoed information with other organizations.** This allows the project to determine for themselves  who, if anyone, needs or gets pre-disclosure information.
+
+## Important considerations
+
+In addition to the requirements detailed below the following should be considered before becoming a CNA:
+
+* **You don't need to become a CNA to get CVEs issued for your project**. Multiple CNAs already cover OSS projects
+  like Red Hat and GitHub. Becoming a new CNA should only be considered if the existing CNAs don't meet
+  the needs of the project.
+* **Becoming a CNA adds a new commitment.** You must have the time and knowledge necessary to implement CNA processes.
+  Being a CNA is an ongoing commitment so your project should have multiple people able to manage the CNA and plans for continuity.
+* **Issuing CVEs is the most important role of a CNA.** If you don't plan on issuing CVEs then becoming a CNA is not necessary.
```

Comment:
what about "right of refusal", so a project can prevent false CVEs from being issued?

Reply:
There is no absolute right-to-refusal being a CNA, you're only guaranteed the first-decision on the report. If the reporter disagrees with the determination they can appeal to your root CNA (usually MITRE or Red Hat) but at that point you're involved in that conversation too. This has a preventative effect because the whole reason false CVEs are being issued at all is root CNAs don't always have the expertise to decide vuln/no vuln so air on the side of caution and issue a CVE ID. With experts involved in the process that can't happen.

Reply:
Awesome - that sounds like a very good reason to become a CNA even if you plan on issuing no CVEs, which contradicts this line :-)